torch/backends/cudnn/rnn.py

```diff
@@ -190,7 +196,11 @@ def forward(fn, input, hx, weight, output, hy):
         handle = cudnn.get_handle()
         fn.datatype = cudnn._typemap[input.type()]
         is_input_packed = fn.batch_sizes is not None
-
+        if is_input_packed:
+            warnings.warn(
+                "Warning: persistent algorithm not supported for variable length input."
+                "Switching to standard")
+            fn.persistent = False  # persistent algo is not supported for variable length input
```

Comment:
Damn so even slight changes in the sequence length (e.g. from a power of 2 to sth else) can have a dramatic effect on the perf? Is there really no way to predict that? Maybe we should just roll out our own benchmarks and hardcode the results... How frequent are cuBLAS releases?